toolchain/semantics/semantics_ir_factory.cpp

```diff
@@ -165,91 +65,210 @@ static auto GetBinaryOp(TokenKind kind) -> Semantics::BinaryOperator::Op {
   }
 }
 
-void SemanticsIRFactory::TransformInfixOperator(
-    llvm::SmallVector<Semantics::NodeRef, 0>& nodes, ParseTree::Node node,
-    Semantics::NodeId target_id) {
-  CARBON_CHECK(parse_tree().node_kind(node) == ParseNodeKind::InfixOperator());
-
-  auto token = parse_tree().node_token(node);
-  auto token_kind = tokens_->GetKind(token);
-  auto op = GetBinaryOp(token_kind);
-
-  auto rhs_id = next_id();
-  auto lhs_id = next_id();
-  nodes.push_back(semantics_.nodes_.Store(
-      Semantics::BinaryOperator(node, target_id, op, lhs_id, rhs_id)));
-  auto subtree = ParseSubtreeConsumer::ForParent(parse_tree(), node);
-  TransformExpression(nodes, subtree.RequireConsume(), rhs_id);
-  TransformExpression(nodes, subtree.RequireConsume(), lhs_id);
-}
-
-/*
-auto SemanticsIRFactory::TransformParameterList(ParseTree::Node node)
-    -> llvm::SmallVector<Semantics::PatternBinding, 0> {
-  CARBON_CHECK(parse_tree().node_kind(node) ==
-ParseNodeKind::ParameterList());
-
-  auto subtree = ParseSubtreeConsumer::ForParent(parse_tree(), node);
-  RequireNodeEmpty(subtree.RequireConsume(ParseNodeKind::ParameterListEnd()));
-
-  llvm::SmallVector<Semantics::PatternBinding, 0> params;
-  if (auto first_param_node =
-          subtree.TryConsume(ParseNodeKind::PatternBinding())) {
-    params.push_back(TransformPatternBinding(*first_param_node));
-
-    while (auto comma_node =
-               subtree.TryConsume(ParseNodeKind::ParameterListComma())) {
-      RequireNodeEmpty(*comma_node);
-      params.push_back(TransformPatternBinding(
-          subtree.RequireConsume(ParseNodeKind::PatternBinding())));
+// bazel run //toolchain/driver:carbon dump semantics-ir
+// testdata/function/basic.carbon
+void SemanticsIRFactory::Build() {
+  // Silence "unused" build warning.
```

Comment:
I don't understand this comment in this context.

Reply:
Removed, debugging leftover.